from .ResultException import ResultException


class UnwrapFailedException(ResultException): ...
